csrankings-x.csv

```diff
@@ -216,6 +216,7 @@ Xiaong-Dong Li,RMIT University,https://titan.csit.rmit.edu.au/~e46507,AQewL04AAA
 Xiaonan Guo,George Mason University,https://mason.gmu.edu/~xguo8/,4fkyrCIAAAAJ
 Xiaonan Zhang,Florida State University,https://www.cs.fsu.edu/~xzhang/index.html,i2zf_1QAAAAJ
 Xiaoning Ding,NJIT,https://web.njit.edu/~dingxn,2lfRQF8AAAAJ
+Xiaoning Du,Monash University,https://xiaoningdu.github.io/,TtA9_B0AAAAJ

```

Comment:
Needs disambiguation suffix (see https://dblp.org/pid/97/6476-1.html).

Reply:
The disambiguation suffix was added in the commit below. Sorry for the neglect.